app/bundles/EmailBundle/Mailer/Message/MauticMessage.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Mautic\EmailBundle\Mailer\Message;
+
+use Symfony\Component\Mime\Email;
+
+class MauticMessage extends Email
```

Comment:
@escopecz @RCheesley before the refactor there were references to `Email` or `SwiftMessage`, from now on we should only use `MauticMessage`

Reply:
I think this will mean quite the refactor of the dev docs!